sumo/plotting/optics_plotter.py

```diff
@@ -204,9 +212,15 @@ def get_plot(
             range(n_plots), self._spec_data.items(), ymin_series, ymax_series
         ):
             ax = fig.axes[i]
-            _plot_spectrum(data, self._label, self._band_gap, ax, optics_colours)
-
-            xmax = xmax if xmax else self._xmax
+            _plot_spectrum(data, self._label, self._band_gap, ax, optics_colours, units)
+
+            if units in ['ev', 'eV'] and not xmax:
+                xmax = self._xmax  # use sumo-determined energy limits
+            elif units == 'nm':
+                xmax = xmax if xmax else 2500  # use default minimum energy (max wavelength) of
```

Comment:
Can you change this to ` xmax = xmax if xmax is not None else 2500` as otherwise if a user sets xmax = 0 it will get changed to 2500.

Reply:
Done!